public/main.js

```diff
@@ -1,14 +1,15 @@
 var startLaunchTime = Date.now();
+var winston = require('../source/utils/winston');
```

Comment:
Is winston used in this file or just imported for side effects?

Reply:
Currently only for the side effects.